### ISSUE TITLE
Data sources UI

### ISSUE THREAD

**Issue by bkapourani** [LinDA-tools/LindaWorkbench#154]
Please see the attached file for a minor "nice to have" feature, if you all agree.

![datasourceui](https://cloud.githubusercontent.com/assets/11331158/6708422/cd8cf34e-cd78-11e4-8c42-12870ec51719.png)

**Comment by smouzakitis** [comment]
Hi Barbara, i am not sure about filling empty space with text. Maybe it is better not to overload every page with much information. We can just have pop-up or tooltips as you are proposing?

**Comment by bkapourani** [comment]
I agree - I do not like either too much text; however, we should make the user fill confident on the step he/she takes and to know each time where he/she stands in the platform. This is the big idea; so, we can go for the tooltips, but then I would prefer the alignment of the page content in the middle... not leaving so much white, useless place in the right. Do you agree?
